test/libscap/test_suites/engines/kmod/kmod.cpp

```diff
@@ -193,8 +193,8 @@ TEST(kmod, metrics_v2_check_per_CPU_stats)
 
 	ssize_t num_online_CPUs = sysconf(_SC_NPROCESSORS_ONLN);
 
-	// We want to check our CPUs counters
-	uint32_t flags = METRICS_V2_KERNEL_COUNTERS;
+	// Enabling `METRICS_V2_KERNEL_COUNTERS_PER_CPU` we also enable `METRICS_V2_KERNEL_COUNTERS`
```

Comment:
we should definitely unify the test for the 3 engines in some way because we are copying and pasting the same code 3 times for all the tests, in the end, the interface is the same... BTW I'm not doing it in this PR :/

Reply:
Agree!